elementemplater-class.php

```diff
@@ -60,6 +61,18 @@ public function enqueue_scripts() {
 			wp_enqueue_script( 'builder-fullwidth-js', plugins_url( 'assets/custom.js', __FILE__ ), array( 'jquery' ), '', true );
 		}
 	}
+
+	public function simple_notice() {
+		$now        = strtotime( 'now' );
+		$start_date = strtotime( '3 december 2018 00:00' );
+		if ( $now < $start_date ) {
+			return;
+		}
+		echo '<div class="updated notice is-dismissible">
```

Comment:
Isn't it handled automagically by WP ?
checked this post: https://digwp.com/2016/05/wordpress-admin-notices/

Reply:
Unfortunately it isn't 😞 .

From the WordPress [Plugins API](https://codex.wordpress.org/Plugin_API/Action_Reference/admin_notices): 

```The class name is-dismissible will automatically trigger a closing icon to be added to your message via JavaScript. Its behavior, however, applies only on the current screen. It will not prevent a message from re-appearing once the page re-loads, or another page is loaded.```

Reply:
hmmm ok, I'll add/remove it with options